fxmanifest.lua

```diff
@@ -7,7 +7,8 @@ version '1.0.0'
 shared_scripts {
     'config.lua',
     '@qb-core/shared/locale.lua',
-    'locales/en.lua', -- Change to the language you want
+    'locales/en.lua',
```

Comment:
I would remove this line since it is already being included by 'locales/*.lua'.

Reply:
This part ***must not*** be removed. In order for locales to use English as the fallback language, it must be loaded first, so the others can use it. `locales/*` does not guarantee loading `locales/en.lua` first.